npm/@fastly/cli/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@fastly/cli",
+  "version": "10.12.3",
+  "description": "Build, deploy and configure Fastly services from your terminal",
+  "type": "module",
+  "scripts": {
+    "version": "node ./update.js $npm_package_version"
+  },
+  "devDependencies": {
+    "decompress": "^4.2.1",
+    "decompress-targz": "^4.1.1"
+  },
+  "main": "index.js",
+  "engines": {
+    "node": ">=16"
+  },
+  "bin": {
+    "fastly": "./fastly.js"
+  },
+  "optionalDependencies": {
+    "@fastly/cli-darwin-arm64": "10.12.3",
+    "@fastly/cli-darwin-x64": "10.12.3",
+    "@fastly/cli-linux-386": "10.12.3",
+    "@fastly/cli-linux-arm64": "10.12.3",
+    "@fastly/cli-linux-x64": "10.12.3",
+    "@fastly/cli-win32-386": "10.12.3",
+    "@fastly/cli-win32-arm64": "10.12.3",
+    "@fastly/cli-win32-x64": "10.12.3"
```

Comment:
Can these be autogenerated in some way? This seems like a future failure point when someone forgets to update them.

Reply:
Now auto-generated.